build.gradle

```diff
@@ -0,0 +1,37 @@
+plugins {
+	id 'java'
+	id 'org.springframework.boot' version '2.6.7'
+	id 'io.spring.dependency-management' version '1.1.0'
+	id 'jacoco'
+}
+
+group = 'com.techsophy'
+version = '0.0.1-SNAPSHOT'
+compileJava {
+	sourceCompatibility = 11
+	targetCompatibility = 11
+}
+
+configurations {
+	compileOnly {
+		extendsFrom annotationProcessor
+	}
+}
+
+repositories {
+	mavenCentral()
+}
+
+dependencies {
+	implementation 'org.springframework.boot:spring-boot-starter-webflux'
+	compileOnly 'org.projectlombok:lombok'
+	developmentOnly 'org.springframework.boot:spring-boot-devtools'
+	annotationProcessor 'org.projectlombok:lombok'
+	implementation 'org.springframework.boot:spring-boot-starter-test'
+	testImplementation group: 'org.mockito', name: 'mockito-inline', version: '2.7.21'
+	testImplementation group: 'com.github.tomakehurst', name: 'wiremock-jre8', version: '2.27.2'
+}
+
```

Comment:
coverage check needs to be added

Reply:
added coverage of minimum 85% in build.gradle